documentation/book/proc-configuring-clients-to-trust-cluster-ca.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='configuring-clients-to-trust-cluster-ca-{context}']
+= Configuring clients to trust the Cluster CA 
+
+If a Kafka client wants to connect to the `tls` listener on port 9093 or the `external` listener on port 9094, it needs to trust the cluster CA certificate.
+
+.Procedure
+
+. The cluster CA certificate can be extracted from the generated `_<cluster-name>_-cluster-ca-cert` `Secret`.
++
+ifdef::Kubernetes[]
+On {KubernetesName}, the certificate can be extracted with the following command:
++
+[source,shell]
+kubectl get secret _<cluster-name>_-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
+endif::Kubernetes[]
+On {OpenShiftName}, the certificate can be extracted with the following command:
++
+[source,shell]
```

Comment:
You need subs="+quotes" in your [source] block for replaceables to render properly - also you need to remove 'shell' - for some reason that causes errors.  All of this here: http://strimzi.io/contributing/guide/#replaceables